manual/src/server/sal.md

```diff
@@ -0,0 +1,789 @@
+- transfers a fungible token from the given address to the given target address
+
+```
+func (c *Client) SafeTransferFungible(args TransferFungible) (string, error)
+```
+
+### TransferFungible
+- transfers the given fungible token from the given address to the given target address
+
+```
+func (c *Client) TransferFungible(args TransferFungible) (string, error)
+```
+
+### SetFungibleApproval
+- approves the given address to spend the given tokenId of the given fungible token
+
+```
+func (c *Client) SetFungibleApproval(args SetFungibleApproval) (string, error)
+```
+
+### SetFungibleApprovalForAll
+- approves the given address to spend all the given fungible tokens
+
+```
+func (c *Client) SetFungibleApprovalForAll(args SetFungibleApprovalForAll) (string, error)
+```
+
+### GetApprovalForFungible
+- returns whether the given address is approved to spend the given tokenId of the given fungible token
+
+```
+func (c *Client) GetApprovalForFungible(args ApprovalForFungible) (bool, error)
+```
+
+### GetApprovalForAllFungible
+- returns whether the given address is approved to spend all the given fungible tokens
+
+```
+func (c *Client) GetApprovalForAllFungible(args ApprovalForFungible) (bool, error)
+```
+
+### GetMultisigOwners
+- fetches the owner addresses for a multisig contract
+
+```
+func (c *Client) GetMultisigOwners(contractAddress string) ([]string, error)
+```
+
+### GetMultisigThreshold
+- fetches the treshold for a multisig contract
+
+```
+func (c *Client) GetMultisigThreshold(contractAddress string) (string, error)
+```
+
+### AddMultisigOwner
+- adds an owner to a multisig contract
+
+```
+func (c *Client) AddMultisigOwner(args MultisigOwner) (string, error)
+```
+
+### RemoveMultisigOwner
+- adds an owner to a multisig contract
+
+```
+func (c *Client) RemoveMultisigOwner(args MultisigOwner) (string, error)
+```
+
+### ApproveHash
+- approves a transaction hash
+
+```
+func (c *Client) ApproveHash(args ApproveHash) (string, error)
+```
+
+### IsApproved
+- approves a transaction hash
+
+```
+func (c *Client) IsApproved(args ApproveHash) (bool, error)
+```
+
+### InitiateMultisigEthTransfer
+- initiates a multisig eth transfer operation
+
+```
+func (c *Client) InitiateMultisigEthTransfer(args InitiateMultisigEthTransfer) (string, error)
+```
+
+### InitiateMultisigTokenTransfer
+- initiates a multisig eth transfer operation
+
+```
+func (c *Client) InitiateMultisigTokenTransfer(args InitiateMultisigTokenTransfer) (string, error)
+```
+
+## Nostr
+
+### Load
+- loads key
+- starts session and creates state for client
+
+```
+func (c *Client) Load(args Load) error
+```
+
+### ID
+- returns the nostr ID for the client
+
+```
+func (c *Client) GetId() (string, error)
+```
+
+### GetPublicKey
+- returns the public key of the client in hex
+
+```
+func (c *Client) GetPublicKey() (string, error)
+```
+
+### ConnectAuthRelay
+- connects to an authenticated relay with a given url
+
+```
+func (c *Client) ConnectAuthRelay(url string) error
```

Comment:
I think should be `ConnectToRelay` or `ConnectToAuthtenticatedRelay` because now it felt like it connects and authenticate the user, not being connected to authenticated relay, which i also don't understand